### ISSUE TITLE
[FEATURE] exclude post under certain upvote threshold 

### ISSUE THREAD

**Issue by vanharen07** [aliparlakci/bulk-downloader-for-reddit#610]
- [ ] I am requesting a feature.
- [ ] I am running the latest version of BDfR
- [ ] I have read the [Opening an issue](../../README.md#configuration)
  
## Description
An argument to exclude posts under a certain number of upvotes. This will help to try and weed out the lower effort/quality content when downloading a lot of posts at the same time. As when downloading a thousand posts you in some situations only want the higher upvoted content. 

Don’t know if it’s possible, but thank you anyways for your hard work.

**Comment by Serene-Arc** [comment]
I would be able to implement this without issue but, to be sure, you understand that this won't increase the number of submissions retrieved? It'll still be at the 1000 post maximum

**Comment by vanharen07** [comment]
Yes I totally understand. I maybe wasn’t clear enough it is ti just sort out the lower quality content from new as i have a command that loops when its done.

**Comment by Serene-Arc** [comment]
Any thoughts @aliparlakci

**Comment by Fakeaccount12312** [comment]
As a workaround, I changed the file naming scheme to start with the number of upvotes with 
`--file-scheme` "{UPVOTES}__splitter__{TITLE}-{POSTID}"
Then I sorted the files by number in the explorer, selected and deleted all under the threshold and removed the "__splitter__" with a renaming tool. These steps could be automated easily too. It doesn't save bandwith as you still download them all, but storage.
